cmd/policy/policy.go

```diff
@@ -50,6 +53,25 @@ func CreateCommand() *cobra.Command {
 	return c
 }
 
+func OpalConvertCommand() *cobra.Command {
+	// only available for opal
+	c := &cobra.Command{
+		Use:   "convert",
+		Short: "Convert opal built-in policies",
```

Comment:
This is basically converting the rego to metadata right.. maybe the short needs to be more clear.. I was confused on what it does in the first go..

Reply:
I'll make it more descriptive, basically rego_doc to metadata and restructures the dir structure into the lacework structure